chemberta/masked-lm/train_roberta_mlm.py

```diff
@@ -29,11 +31,12 @@
 FLAGS = flags.FLAGS
 
 # RobertaConfig params
-flags.DEFINE_integer(name="vocab_size", default=52_000, help="")
-flags.DEFINE_integer(name="max_position_embeddings", default=512, help="")
-flags.DEFINE_integer(name="num_attention_heads", default=12, help="")
-flags.DEFINE_integer(name="num_hidden_layers", default=6, help="")
+flags.DEFINE_integer(name="vocab_size", default=600, help="")
```

Comment:
Either works but maybe use 512?

Reply:
unfortunately it needs to be more than 512